src/Avalonia.Visuals/Platform/IPlatformColorSchemeProvider.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Avalonia.Media;
+
+namespace Avalonia.Platform
+{
+    public interface IPlatformColorSchemeProvider
+    {
+        public Color GetSystemAccentColor(Color fallBackColor);
```

Comment:
Do we need to pass fallback color, when we can return nullable or make it TryGet pattern?

Reply:
I'm working on that